src/components/ViewRequest/DetailSections/DetailsTemplate.tsx

```diff
@@ -229,6 +230,30 @@ const DetailsTemplate = ({
                           Edit
                         </Button>
                       )}
+                    {isEditable &&
+                      isEditor &&
+                      setEditSection &&
+                      setIsEditOpen && (
+                        <Button
+                          icon={<DeleteIcon />}
+                          aria-label="Delete"
+                          disabled={updateRequest.isLoading}
```

Comment:
Should this button also be disabled if there aren't anymore methods on the request, since it is required to have one?